---
title: "MICB425 Portfolio - Microbial Ecological Genomics: From Genomes to Biomes, Spring 2018"
author: "Cheng Ching, May, Ho, 31824139"
date: "version `r format(Sys.time(), '%B %d, %Y')`"
output: 
  html_document:
    toc: true
---
# Module 01
Reserve the first level headings (#) for the start of a new Module. This will help to organize your portfolio in an intuitive fashion.  
**Note: Please edit this template to your heart's content. This is meant to be the armature upon which you build your individual portfolio. You do not need to keep this instructive text in your final portfolio, although you do need to keep module and assignment names so we can identify what is what.**



## Module 01 Portfolio Content

* Evidence worksheet_01
    + Completion status: X
    + Comments:
* Evidence worksheet_02
    + Completion status: X
    + Comments:
* Evidence worksheet_03
    + Completion status: X
    + Comments:
* Problem Set_01
    + Completion status: X
    + Comments:
* Problem Set_02
    + Completion status: X
    + Comments:
* Writing assessment_01
    + Completion status:
    + Comments:
* Additional Readings
    + Completion status:
    + Comments: Need links.  



## Data science Friday  

### Data Science

* Installation check
    + Completion status: X
    + Comments:
* Portfolio repo setup
    + Completion status: X
    + Comments:
* RMarkdown Pretty PDF Challenge
    + Completion status: X
    + Comments:
* ggplot
    + Completion status: 9/10
    + Comments: 
    + Exercise 3 - check your plot title. Are you showing phyla?
    + Exercise 4 - explore the 'scales' parameter within facet to allow the y axes to change scale in the separate facets.
The remaining second level headers (##) are for separating data science Friday, regular course, and project content. In this module, you will only need to include data science Friday and regular course content; projects will come later in the course.

### Installation check
Third level headers (###) should be used for links to assignments, evidence worksheets, problem sets, and readings, as seen here.  

Use this space to include your installation screenshots.



### Portfolio repo setup
Detail the code you used to create, initialize, and push your portfolio repo to GitHub. This will be helpful as you will need to repeat many of these steps to update your porfolio throughout the course. 

In Git: 
mkdir MICB425_portfolio 

cd MICB425_portfolio 

cd MICB425_portfolio 

Create repository on GitHub page. 

git init 

git add . 

git commit -m "First commit" 

git remote add origin https://remote_repository_URL 

git remote -v 

git push -u origin master 


### RMarkdown pretty html challenge
Paste your code from the in-class activity of recreating the example html.

```{r child = "DS_assignment3.Rmd"}
```

## Origins and Earth Systems
### Evidence worksheet 01 
The template for the first Evidence Worksheet has been included here. The first thing for any assignment should link(s) to any relevant literature (which should be included as full citations in a module references section below).

You can copy-paste in the answers you recorded when working through the evidence worksheet into this portfolio template. 

As you include Evidence worksheets and Problem sets in the future, ensure that you delineate Questions/Learning Objectives/etc. by using headers that are 4th level and greater. This will still create header markings when you render (knit) the document, but will exclude these levels from the Table of Contents. That's a good thing. You don't' want to clutter the Table of Contents too much.

[Whitman *et al* 1998](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)

#### Learning objectives
Describe the numerical abundance of microbial life in relation to ecology and biogeochemistry of Earth systems. 

#### General questions
* What were the main questions being asked?  
What is the abundance of prokaryotes on earth? 
What is the total amount of cellular carbon produced by these prokaryotes on earth?  

* What were the primary methodological approaches used?  
To count prokaryotes 
	- aquatic environments: used cellular density 
	- soil: direct counts from a coniferous forest ultisol (cells/g) 
		-  unpublished field studies of E. A. Paul for cultivated soils 
		- terrestrial subsurface 
		- unconsolidated sediments represent most of marine subsurface and have been determined 
		- assuming that average porosity of terrestrial subsurface is 3%
		- estimation from groundwater data based on values from seven sites and four studies   

Other Habitats: 
	- animals 
		- human: cell density of prokaryotes on the skin multiply by skin surface area 
		- insects like termite by counting number of insect and number of prokaryotes in said  insect 
		- leaves: can be estimated by assuming a dense population and high leaf area index  
		- air: pre-calculated 
Carbon Content: 
	- estimated from cell numbers in soil, aquatic systems, and the subsurface 
		-  cellular carbon is assumed to be one-half of dry weight for soil and subsurface 
		- take average dry weight of prokaryotic cells  multiple by number of cell 
	- aquatic systems: assumed that average cellular carbon for sedimentary and planktonic  prokaryotes to be 10 and 20 fg of C/cell respectively then multiple that with number of cells in aquatic systems  

* Summarize the main results or findings.  
	- Total number of prokaryotes is 4-6 x 1030 cells and 350-500 Pg of C (1 Pg = 1015 g) 
	- Represent the largest pool of nutrients such as N and P 
Essentially, prokaryotic biomass as a major contributor to total biosphere 

* Do new questions arise from the results? 
	- what is the genetic diversity of these prokaryotes?  
	- The number of prokaryotic species? 
How does prokaryotic turnover affect carbon fixation and carbon cycle? 

* Were there any specific challenges or advantages in understanding the paper (*e.g.* did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)?  
	- bombards you with numbers  
	- inadequate explanation of some assumptions made especially when estimating carbon content 
	- can be found in literature however 
	- assumed that the papers they cited had the proper methods  
	- lots of estimation of cell densities 

```{r child = "Evidence_worksheet_02.Rmd"}
```

```{r child = "Evidence_worksheet_03.Rmd"}
```

### Problem set 01
#### Learning objectives: 
Describe the numerical abundance of microbial life in relation to the ecology and biogeochemistry of Earth systems.

#### Specific questions:
* What are the primary prokaryotic habitats on Earth and how do they vary with respect to their capacity to support life? Provide a breakdown of total cell abundance for each primary habitat from the tables provided in the text. 

		a. Aquatic : 1.18 x 10^29^
		b. Soil: 2.556 x 10^29^
    c. subsurface: 3.8 x 10^30^  

* What is the estimated prokaryotic cell abundance in the upper 200 m of the ocean and what fraction of this biomass is represented by marine cyanobacterium including Prochlorococcus? What is the significance of this ratio with respect to carbon cycling in the ocean and the atmospheric composition of the Earth? 
3.6 x 10^28^ 
cyanobacteria: 4x 10^4^ cells/ml / 5 x 10^5^ cells x 100 = 8%  

* What is the difference between an autotroph, heterotroph, and a lithotroph based on information provided in the text?  
		a. autotroph: "self-nourishing" fix inorganic carbon (CO2) -> biomass 
		b. heterotroph: assimilate organic carbon 
    c. lithotroph: use inorganic substances  

* Based on information provided in the text and your knowledge of geography what is the deepest habitat capable of supporting prokaryotic life? What is the primary limiting factor at this depth? 

Since the temperature drop is 22 degrees drop per km so the deepest part that can support life is Mariana Trench 10.9km + plus an extra 5 km  

* Based on information provided in the text your knowledge of geography what is the highest habitat capable of supporting prokaryotic life? What is the primary limiting factor at this height?  
22 km on top of the 8.8 km on Mt. Everest. A limiting factor at that height would be obtaining enough nutrients. 

* Based on estimates of prokaryotic habitat limitation, what is the vertical distance of the Earth's biosphere measured in km?  

  -22 + 8.8 + 10.9 + 5 = 46.7 km

* How was annual cellular production of prokaryotes described in Table 7 column four determined? (Provide an example of the calculation)  
		- 3.6 x 1028 / 16 x 365 = 8.4 x 1029 
		- population size divided by turnover time per day times 365 days  

* What is the relationship between carbon content, carbon assimilation efficiency and turnover rates in the upper 200m of the ocean? Why does this vary with depth in the ocean and between terrestrial and marine habitats?  
    - carbon efficiency is 20%  
    - 5-20 fg C/cell 
    - (3.6 x 10^26^ Pg/cell)(20x10^30^ cell) = 0.72 Pg of C in marine heterotrophs  

* How were the frequency numbers for four simultaneous mutations in shared genes determined for marine heterotrophs and marine autotrophs given an average mutation rate of 4 x 10^-7^ per DNA replication? (Provide an example of the calculation with units. Hint: cell and generation cancel out)  
    - 4 x 10^-7^ mutations/generation
    -(4 x 10^-7-^)^4^ = 2.56 x 10^-26^ mutations/generation
    - 365/16 = 22.5 turnovers/yr
    -(3.1 x 10^28^ cells) x 22.5 = 8.2 x 10^29^ cells/yr
    -(8.2 x 10^29^ cells/yr)(2.56 x 10^-26^ mutations/generation) = 2.1 x 10^4^ mutations/yr  
    
* Given the large population size and high mutation rate of prokaryotic cells, what are the implications with respect to genetic diversity and adaptive potential? Are point mutations the only way in which microbial genomes diversify and adapt?  

 Prokaryotes would have high genetic diversity and the ability to adapt quickly dude to their high mutation rate. Insertions and deletions are generally detrimental to a gene's function since they shift the reading frame so point mutations tend to be the most common, but there's potential for these type of mutations to promote genetic diversity. 

* What relationships can be inferred between prokaryotic abundance, diversity, and metabolic potential based on the information provided in the text?   

High prokaryotic abundance encourages the diversification of metabolic capabilities in prokaryotes. There are more likely to be more mutations taking place in a larger population of prokaryotes that allow them to fully take advantage of their environment and compete for different resources. 

```{r child = "Problem_set_02.Rmd"}
```

```{r child = "essay_1.Rmd"}
```

### Module 01 references
Utilize this space to include a bibliography of any literature you want associated with this module. We recommend keeping this as the final header under each module.

An example for Whitman and Wiebe (1998) has been included below.  

1. 	Achenbach J. 2012. Spaceship Earth: a new view of environmentalism. Washington Post 2–5.
(https://www.washingtonpost.com/national/health-science/spaceship-earth-a-new-view-of-environmentalism/2011/12/29/gIQAZhH6WP_story.html)
2. 	Canfield DE, Glazer AN, Falkowski PG. 2010. The evolution and future of earth’s nitrogen cycle. Science (80- ) 330:192–196.
[PMID20929768](https://www.ncbi.nlm.nih.gov/pubmed/20929768)
3. 	Falkowski PG, Fenchel T, Delong EF. 2008. The microbial engines that drive earth’s biogeochemical cycles. Science (80- ) 320:1034–1039.
[PMID18497287](https://www.ncbi.nlm.nih.gov/pubmed/18497287)
4. 	Kasting JF, Siefert JL. 2002. Life and the evolution of earth's atmosphere. Library (Lond) 296:1066–1069.
[PMID12004117](https://www.ncbi.nlm.nih.gov/pubmed/12004117)
5. 	Leopold AC, Pickett STA, Ostfeld RS, Shachak M, Likens GE. 1949. The land ethic of Aldo Leopold. J For 5–7.
6. 	Nisbet EG, Sleep NH. 2001. The habitat and nature of early life. Nature 409:1083–1091.
[PMID11234022](https://www.ncbi.nlm.nih.gov/pubmed/11234022)
7. 	Falkowski P, Scholes RJ, Boyle E, Canadell J, Canfield D, Elser J, Gruber N, Hibbard K, Hogberg P, Linder S, Mackenzie FT, Moore B, Pedersen T, Rosental Y, Seitzinger S, Smetacek V, Steffen W. 2000. The global carbon cycle: A test of our knowledge of earth as a system. Science (80- ) 290:291–296.
[PMID11030643](https://www.ncbi.nlm.nih.gov/pubmed/11030643)
8. 	Kallmeyer J, Pockalny R, Adhikari RR, Smith DC, D’Hondt S. 2012. Global distribution of microbial abundance and biomass in subseafloor sediment. Proc Natl Acad Sci 109:16213–16216.
[PMC3479597](https://www.ncbi.nlm.nih.gov/pubmed/22927371)
9. 	Rockström J, Steffen W, Noone K, Persson Å, Chapin FS, Lambin E, Lenton TM, Scheffer M, Folke C, Schellnhuber HJ, Nykvist B, de Wit CA, Hughes T, van der Leeuw S, Rodhe H, Sörlin S, Snyder PK, Costanza R, Svedin U, Falkenmark M, Karlberg L, Corell RW, Fabry VJ, Hansen J, Walker B, Liverman D, Richardson K, Crutzen P, Foley J. 2009. Planetary boundaries: Exploring the safe operating space for humanity. Ecol Soc 14.
[PMID19779433](https://www.ncbi.nlm.nih.gov/pubmed/19779433)
10. Shrag, Geobiology of Anthropocene_2012.pdf.  
11. Breitburg D, Levin LA, Oschlies A, Grégoire M, Chavez FP, Conley DJ, Garçon V, Gilbert D, Gutiérrez D, Isensee K, Jacinto GS, Limburg KE, Montes I, Naqvi SWA, Pitcher GC, Rabalais NN, Roman MR, Rose KA, Seibel BA, Telszewski M, Yasuhara M, Zhang J. 2018.Declining oxygen in the global ocean and coastal waters. Science (80- ) 359.
[PMID29301986](https://www.ncbi.nlm.nih.gov/pubmed/29301986)
12. Waters CN, Zalasiewicz J, Summerhayes C, Barnosky AD, Poirier C, Gałuszka A, Cearreta A, Edgeworth M, Ellis EC, Ellis M, Jeandel C, Leinfelder R, McNeill JR, Richter DDB, Steffen W, Syvitski J, Vidas D, Wagreich M, Williams M, Zhisheng A, Grinevald J, Odada E, Oreskes N, Wolfe AP. 2016. The Anthropocene is functionally and stratigraphically distinct from the Holocene. Science (80- ) 351.
[PMID26744408](https://www.ncbi.nlm.nih.gov/pubmed/26744408)
13. Whitman WB, Coleman DC, Wiebe WJ. 1998. Prokaryotes: The unseen majority. Proc Natl Acad Sci 95:6578–6583.
[PMC33863](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)



# Module 2

## Module 02 Portfolio Content

* Evidence worksheet_04
    + Completion status: X
    + Comments:
* Problem Set_03
    + Completion status: X
    + Comments:
* Writing assessment_02
    + CANCELED
* Additional Readings
    + Completion status:
    + Comments 

```{r child = "Evidence_worksheet_04.Rmd"}
```

```{r child = "Problem_set_03.Rmd"}
```

## Additional Readings
1. 	Madsen EL. 2005. Madsen microbes eco biogeochem process Nature Micro Opinion 3.
[PMID15864265](https://www.ncbi.nlm.nih.gov/pubmed/15864265)

2. 	Martinez A, Bradley AS, Waldbauer JR, Summons RE, DeLong EF. 2007. Proteorhodopsin photosystem gene expression enables photophosphorylation in a heterologous host. Proc Natl Acad Sci 104:5590–5595.
[PMID17372221](https://www.ncbi.nlm.nih.gov/pubmed/17372221)

3. 	Taupp M, Mewis K, Hallam SJ. 2011. The art and design of functional metagenomic screens. Curr Opin Biotechnol 22:465–472.
[PMID21440432](https://www.ncbi.nlm.nih.gov/pubmed/21440432)

4. 	Wooley JC, Godzik A, Friedberg I. 2010. A primer on metagenomics. PLoS Comput Biol 6.
[PMID20195499](https://www.ncbi.nlm.nih.gov/pubmed/20195499)


# Module 3

## Module 03 Portfolio Content

* Evidence worksheet_05
    + Completion status:
    + Comments:
* Problem set_04
    + Completion status:
    + Comments:
* Writing Assessment_03
    + Completion status:
    + Comments:
* Additional Readings
    + Completion status:
    + Comments 
    
### Project 1

* CATME account setup and survey
    + Completion status: X
    + Comments: 
* CATME interim group assessment
    + Completion status: X
    + Comments:
* Project 1
    + Report (80%): 
    + Participation (20%):

```{r child = "Evidence_worksheet_05.Rmd"}
```

```{r child = "Problem_set_04.Rmd"}
```

```{r child = "essay_2.Rmd"}
```

##Project 1

```{r}
htmltools::tags$iframe(title="MICB425 Project 1", src="Proj1.html", height=1000, width=1000)
```

## Additional Readings
1. 	Callahan BJ, McMurdie PJ, Holmes SP. 2017. Exact sequence variants should replace operational taxonomic units in marker-gene data analysis. ISME J 11:2639–2643.
[PMID28731476](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5702726/)

2. 	Gaudet AD, Ramer LM, Nakonechny J, Cragg JJ, Ramer MS. 2010. Small-group learning in an upper-level university biology class enhances academic performance and student attitudes toward group work. PLoS One 5.
[PMID21209910](https://www.ncbi.nlm.nih.gov/pubmed/21209910)

3. 	Hallam SJ, Torres-Beltrán M, Hawley AK. 2017. Monitoring microbial responses to ocean deoxygenation in a model oxygen minimum zone. Sci Data 4:1–3.
[PMID29087370](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5663219/)

4. 	Hawley AK, Brewer HM, Norbeck AD, Pa a-Toli  L, Hallam SJ. 2014. Metaproteomics reveals differential modes of metabolic coupling among ubiquitous oxygen minimum zone microbes. Proc Natl Acad Sci 111:11395–11400.
[PMID25053816](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4128106/)

5. 	Kunin V, Engelbrektson A, Ochman H, Hugenholtz P. 2010. Wrinkles in the rare biosphere: Pyrosequencing errors can lead to artificial inflation of diversity estimates. Environ Microbiol 12:118–123.
[PMID19725865](https://www.ncbi.nlm.nih.gov/pubmed/19725865)

6. 	Cordero OX, Ventouras L-A, DeLong EF, Polz MF. 2012. Public good dynamics drive evolution of iron acquisition strategies in natural bacterioplankton populations. Proc Natl Acad Sci 109:20059–20064.
[PMID23169633](https://www.ncbi.nlm.nih.gov/pubmed/23169633)

7. 	Giovannoni SJ. 2012. Vitamins in the sea. Proc Natl Acad Sci 109:13888–13889.
8. 	Lundin D, Severin I, Logue JB, Östman Ö, Andersson AF, Lindström ES. 2012. Which sequencing depth is sufficient to describe patterns in bacterial alpha- and beta-diversity? Environ Microbiol Rep 4:367–372.
[PMID23760801](https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1758-2229.2012.00345.x)

9. 	Morris JJ, Lenski RE, Zinser ER. 2012. The Black Queen Hypothesis: Evolution of Dependencies through Adaptative Gene Loss. MBio 3:1–7.
[PMID22448042](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3315703/)

10. Thompson JR, Pacocha S, Pharino C, Klepac-Ceraj V, Hunt DE, Benoit J, Sarma-Rupavtarm R, Distel DL, Polz MF. 2005. Genotypic diversity within a natural coastal bacterioplankton population. Science (80- ) 307:1311–1313.
[PMID15731455](https://www.ncbi.nlm.nih.gov/pubmed/15731455)

11.	Welch RA, Burland V, Plunkett G, Redford P, Roesch P, Rasko D, Buckles EL, Liou S-R, Boutin A, Hackett J, Stroud D, Mayhew GF, Rose DJ, Zhou S, Schwartz DC, Perna NT, Mobley HLT, Donnenberg MS, Blattner FR. 2002. Extensive mosaic structure revealed by the complete genome sequence of uropathogenic Escherichia coli. Proc Natl Acad Sci U S A 99:17020–4.
[PMID12471157](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC139262/)

12. Torres-Beltrán M, Hawley AK, Capelle D, Zaikova E, Walsh DA, Mueller A, Scofield M, Payne C, Pakhomova L, Kheirandish S, Finke J, Bhatia M, Shevchuk O, Gies EA, Fairley D, Michiels C, Suttle CA, Whitney F, Crowe SA, Tortell PD, Hallam SJ. 2017. A compendium of geochemical information from the Saanich Inlet water column. Sci Data 4:1–10.
[PMID29087371](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5663218/)

13. Welch DBM, Huse SM. 2011. Microbial Diversity in the Deep Sea and the Underexplored “Rare Biosphere.” Handb Mol Microb Ecol II Metagenomics Differ Habitats 243–252.
[PMID16880384](https://www.ncbi.nlm.nih.gov/pubmed/16880384)


# Module 4
## Module 04 Portfolio Content

* CATME final group assessment
    + Completion status: 
    + Comments:
* Project 2
    + Report (80%): 
    + Participation (20%):
    
## Project 2
```{r}
htmltools::tags$iframe(title="MICB425 Project 2", src="Proj2.html", height=1000, width=1000)
```
    

